### ISSUE TITLE
Drop Support For Python 3.5

### ISSUE THREAD

**Issue by jacobdbrown4** [byuccl/spydrnet-tmr#10]
Now that SpyDrNet doesn't support Python 3.5, some SpyDrNet-TMR tests fail for Python 3.5. We should probably just drop support for Python 3.5.

**Comment by wirthlin** [comment]
I agree that we should drop support for Python 3.5 for SpyDrNet-TMR for the next release.

**Comment by jacobdbrown4** [comment]
Python 3.5 is dropped in release 1.2.0
